src/components/InputBox.js

```diff
@@ -1,7 +1,7 @@
 
-function InputBox({placeholder}){
+function InputBox({placeholder, name, type}){
```

Comment:
Lets add default values so in the future it can be easily added without requiring all the fields.

Reply:
I added default values to Inpubox and a AuthenticationForm components